javatests/arcs/core/host/test.arcs

```diff
@@ -5,9 +5,15 @@ particle TestProdParticle in 'arcs.core.host.TestProdParticle'
 
 particle TestHostParticle in 'arcs.core.host.TestHostParticle'
 
+particle TestPlannedParticle in 'arcs.core.host.TestPlannedParticle'
```

Comment:
do you think TestPlannedParticle is a good name? What are you trying to convey? I kind of like "Reflective particle", but I understand that folks assume reflection == Java reflection, but I mean it in a broader sense.  After wikipedia: reflection is the ability of a process to introspect its own structure - so that's what I am trying to convey... though I'm easy.

Reply:
After looking at this code for the past few days, I've warmed up to the name "Reflective" :)